.gitmodules

```diff
@@ -34,3 +34,10 @@
 [submodule "modules/DroneCAN/libcanard"]
 	path = modules/DroneCAN/libcanard
 	url = https://github.com/DroneCAN/libcanard
+[submodule "modules/Micro-XRCE-DDS-Client"]
+	path = modules/Micro-XRCE-DDS-Client
+	url = https://github.com/eProsima/Micro-XRCE-DDS-Client.git
+[submodule "modules/Micro-CDR"]
```

Comment:
do we need the Micro-CDR submodule? I know we may use it in the future, but I don't think we need it now, or have I missed a dependency?

Reply:
I tried removing it and got a billion linker errors. We have includes for `#include "ucdr/microcdr.h"` in `AP_DDS_Client`. That said, this might because we aren't building the micro XRCE Client correctly, since microcdr is a dependency there.